### PR TITLE
This removes usage of DJANGO_ENV (which is generally a less-flexible than the built-in DJANGO_SETTINGS_MODULE)

### PR DESCRIPTION
**Pull request by rosskarchner**
Also, makes the default DJANGO_SETTINGS_MODULE suitable for local development,
cfgov.settings.local

### CODE REVIEW

File: .env_SAMPLE [hunk 1]
```diff
@@ -7,9 +7,6 @@
 # Django Application Server.
 ############################
 
-export DJANGO_ENV=base
-#export DJANGO_STATIC_ROOT=<path_to_static_files>
-
```
Comment (kave) [inline .env_SAMPLE]: still going to need `DJANGO_STATIC_ROOT` for setting the static_url on flapjack/refresh servers

Comment (rosskarchner) [inline .env_SAMPLE]: good catch, restored that line.

**Comment by rosskarchner** [comment]
It seems like the travis failure was a networking failure with npm

**Comment by kave** [comment]
:+1:

**Comment by kurtw** [comment]
Sorry for the late review. This slipped through the cracks.

**Pull request merged**